paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/RollbackToAction.java

```diff
@@ -32,14 +33,18 @@ public class RollbackToAction extends TableActionBase {
 
     private final String version;
 
+    private final Boolean isTimestamp;
```

Comment:
Why not use long timestamp uniformly in flink action && flink procedure && spark procedure?

Reply:
Firstly I also want to change it like you suggested,  but consider older version spark user their procedure sql would not be compatible, so keep it can use in older version first. @wwj6591812